input/ignoreWarnings.txt

```diff
@@ -39,65 +38,40 @@ INFORMATION: Parameters/FindSubjectMolecConseqOutput: Parameters.parameter[1].re
 INFORMATION: Parameters/FindSubjectMolecConseqOutput: Parameters.parameter[2].resource/*Observation/MC3*/.derivedFrom[0]: This element does not match any known slice defined in the profile http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-consequence%
 INFORMATION: Parameters/FindSubjectMolecConseqOutput: Parameters.parameter[3].resource/*Observation/MC4*/.derivedFrom[0]: This element does not match any known slice defined in the profile http://hl7.org/fhir/uv/genomics-reporting/StructureDefinition/molecular-consequence%
 
+# The following code system are not discoverable because they are code systems in THO, but THO doesn't define the codes. They have been checked and the identifiers are correct
```

Comment:
@patrick-werner - This isn't a bad thing, but we originally had this more generic comment, and was asked to be more specific, and to ensure we prove we CSs were in THO, or that we had at least requested them. Still, glad to see this is OK.